### DISCUSSION TITLE
Where are the site branding values stored

### DISCUSSION THREAD

**Discussion by MrRoyce** [mastodon/mastodon#25629] [Q&A]
Hi all, I'm trying to automate setting up mastodon with ansible based on https://github.com/mastodon/mastodon-ansible

Everything is up and running, but I'm trying to see where I can automatically specify the site name, description, etc with a config file or other parameter.  

Is there a way to do that or am I stuck having to use the Admin UI to update the branding information?

Thanks in advance!

**Comment by justdaj** [comment]
Have a look in

`live/.env.production`
